app/routes/exercises.rb

```diff
@@ -66,6 +66,28 @@ class Exercises < Core
         redirect '/dashboard'
       end
 
+      post '/exercises/:key/request_help' do |key|
```

Comment:
We're using hyphens rather than underscores for urls. Also, we're trying to normalize to using resources (nouns) rather than actions (the HTTP verb is the action). A `POST` is a create action, so the resource should be the thing that gets created when you post here. I think that a _request-for-help_ or a _help-request_ make`POST /exercises/:key/help-request`.

We could also look for another word to disambiguate _request_ since this is a very common term when talking about web applications, since it usually means _HTTP request_. So we could use _appeal for help_ or _plea for help_ or something similar.